### PR TITLE
Create releases for Speech, Firestore and Scheduler

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

File: apis/Google.Cloud.Firestore.V1/Google.Cloud.Firestore.V1/Google.Cloud.Firestore.V1.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Version>1.0.0-beta00</Version>
+    <Version>1.0.0-beta16</Version>
```
Comment (amanda-tarafa) [inline apis/Google.Cloud.Firestore.V1/Google.Cloud.Firestore.V1/Google.Cloud.Firestore.V1.csproj]: Should this be beta01?

File: apis/apis.json [hunk 2]
```diff
@@ -353,7 +353,7 @@
     "id": "Google.Cloud.Firestore.V1",
     "productName": "Firestore",
     "productUrl": "https://firebase.google.com",
-    "version": "1.0.0-beta00",
+    "version": "1.0.0-beta16",
```
Comment (amanda-tarafa) [inline apis/apis.json]: beta01?

File: apis/Google.Cloud.Firestore.V1/Google.Cloud.Firestore.V1/Google.Cloud.Firestore.V1.csproj (hunk 1 above)
Comment (jskeet) [reply apis/Google.Cloud.Firestore.V1/Google.Cloud.Firestore.V1/Google.Cloud.Firestore.V1.csproj]: See comment on the JSON file, that this is generated from.

File: apis/apis.json (hunk 2 above)
Comment (jskeet) [reply apis/apis.json]: I thought it simpler to effective "keep going" from where we'd got to with Google.Cloud.Firestore.V1Beta1. That way both Firestore packages have the same version. While that means skipping a load of betas, it's simplest if Google.Cloud.Firestore 1.0.0-betaX depends on Google.Cloud.Firestore.V1 1.0.0-betaX.

**Pull request merged**